src/server/game/Server/WorldSocket.cpp

```diff
@@ -754,7 +754,7 @@ void WorldSocket::HandleAuthSessionCallback(std::shared_ptr<WorldPackets::Auth::
     {
         if (account.BattleNet.LastIP != address)
         {
-            SendAuthResponseError(ERROR_DENIED);
+            SendAuthResponseError(ERROR_RISK_ACCOUNT_LOCKED);
```

Comment:
it does seem like a better choice but you need to triple check it ingame - wow client doesnt handle all of these codes from giant enum

Reply:
Just checked now, it seems to handle that code okay: https://i.imgur.com/KSr8f9H.png

Reply:
iirc that one comes from bnet, you need to disable all ban/lock checks in bnetserver, then try again

Reply:
That's what I had done in the screenshot. Logged into bnet to pass the check. Banned the account while sitting on realmlist, then tried connecting.